### DISCUSSION TITLE
[classic_queue.default_version] How to know that the v2 option is applied?

### DISCUSSION THREAD

**Discussion by jdfresser** [rabbitmq/rabbitmq-server#6779] [Questions] [answered]
Hello,

I tried to activate the functionnality classic_queue.default_version=2, by editing my configuration on a freshly installed 3.11.5-1 standalone server.
Configuration is done via file: /etc/rabbitmq/rabbitmq.config
```
[
  {rabbit, [
    {tcp_listeners, [{"0.0.0.0", 5672}]},
    {vm_memory_high_watermark, 0.4},
    {disk_free_limit, {mem_relative, 1.5}},
    {classic_queue_default_version, 2},
    {loopback_users, []}
  ]}].
```
This file is read by rabbitmq, as can be seen in the logs:
```
2022-12-29 17:32:42.125541+01:00 [info] <0.231.0>  config file(s) : /etc/rabbitmq/rabbitmq.config
```
How can I know that this configuration is well applied on all queues? If I import definitions from another server, how to know that the classic_queue.default_version=2 is applied?
(Note: I do prefer to avoid applying policies)

I do not see it in the UI. 
I tried to create some new queues, and the restarted the rabbitmq-server, then this logs appeared:
```
2022-12-29 17:32:42.952160+01:00 [info] <0.471.0> Converting queue q.storage_service_file.rt in vhost / from v1 to v2 after clean shutdown
2022-12-29 17:32:42.953308+01:00 [info] <0.474.0> Converting queue q.ftp-all in vhost / from v1 to v2 after clean shutdown
2022-12-29 17:32:42.954954+01:00 [info] <0.471.0> Queue q.storage_service_file.rt in vhost / converted 0 total messages from v1 to v2
2022-12-29 17:32:42.955983+01:00 [info] <0.474.0> Queue q.ftp-all in vhost / converted 0 total messages from v1 to v2
```
I then tried to import definitions from another server (3.9 server), and then restarted the rabbitmq-server.
I didn't see any logs of this kind.

Thanks for your help,

**Comment by lukebakken** [comment, marked as answer]
https://www.rabbitmq.com/rabbitmqctl.8.html#environment

You really should be using the modern `rabbitmq.conf` configuration file.

> I then tried to import definitions from another server (3.9 server), and then restarted the rabbitmq-server. I didn't see any logs of this kind.

Since they are brand-new queues there will be nothing to convert, so no logs.

**Reply by jdfresser** [reply]
Thank you! 

> Since they are brand-new queues there will be nothing to convert, so no logs.
Ok, I didn't see it this way, but it's pretty obvious when you write it down .

> You really should be using the modern rabbitmq.conf configuration file.
Thanks for the suggestion, I will.

**Comment by johanrhodin** [comment]
Look at the files, if the files created are `.idx` then it is `v1`, if the are `.qi`/`.qs` they are `v2`
